inlang/source-code/sdk2/src/lix-plugin/inlangLixPluginV1.ts

```diff
@@ -22,6 +22,8 @@ export const inlangLixPluginV1: LixPlugin<{
 	diff: {
 		// TODO does not account for deletions
 		file: async ({ old, neu, path }) => {
+			const diffRunId = new Date().getTime() + "_" + Math.random();
+			console.time(diffRunId + " diff:");
```

Comment:
Scheduling issues?

Reply:
nope - leftover from debugging of commit increment - will remove it again